app/controllers/inventories_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
   def destroy
     @inventory = Inventory.find(params[:id])
     @inventory.destroy
-    redirect_back(fallback_location: root_path)
+    redirect_to root_path
```

Comment:
Please change this line of code to `redirect_back(fallback_location: root_path)`.

The redirect_to method simply redirects the user to the specified URL, in this case, the root path of the application.

The redirect_back method redirects the user to the URL they were visiting before the current action, or to the provided fallback location if there is no previous URL available. This can be useful in situations where the user is coming from multiple possible sources, such as a link in an email or a bookmark, and you want to redirect them back to the previous page regardless of where they came from. Which ensures a better user experience.

Reply:
@Haywayaheadshot thanks, I was testing the redirect and routes and I forgot to change this back. 👍